src/server/config/swagger.json

```diff
@@ -9,6 +9,40 @@
   "host": "",
   "basePath": "/api",
   "paths": {
+    "/days-to-end-batch/{batchId}": {
+      "get": {
+        "tags": ["status box"],
+        "parameters": [
+          {
+            "name": "batchId",
+            "in": "path",
+            "type": "integer",
+            "required": true
+          }
```

Comment:
An object which allows to input the firebase token should also be added to swagger json.something like this:
`{
            "name": "authorization",
            "in": "header",
            "description": "Firebase token",
            "required": true,
            "type": "string",
            "default": "Bearer ENTER_FIREBASE_TOKEN"
          },`